common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -95,27 +143,15 @@ class ConditionalModule(ConditionalFields, XModule):
         'voted': 'voted'  # poll_question attr
     }
 
-    def _get_condition(self):
```

Comment:
I'm nervous about the change in functionality for pre-existing conditional modules. It seems that the current behavior is to find the first conditional attribute, but now it will always look for an attribute called ``correct``. It might be safer to have ``conditional_attr`` be optional, and if it isn't specified then the current behavior is maintained. Since all pre-existing modules won't have this field set, then they will continue to behave as before.